tst/regression/test_suites/advection_convergence/advection_convergence.py

```diff
@@ -58,6 +59,10 @@ def Prepare(self,parameters, step):
 
 
         n_res = len(lin_res)
+        # make sure we can evenly distribute the MeshBlock sizes
+        assert parameters.num_ranks % 2 == 0, "Num ranks must be multiples of 2 for convergence test."
+        # ensure a minimum block size of 4
+        assert lin_res[0] / parameters.num_ranks <= 8, "Use <= 8 ranks for convergence test."
```

Comment:
Do we need similar checks like this in the other regression tests?

Reply:
I don't think so because the other tests are AMR tests that have many MeshBlocks anyway (so unless you try to run the test on, say, more than 100 ranks this should not come up)